docs/ML_DGA.md

```diff
@@ -0,0 +1,175 @@
+
+```yaml
+output.elasticsearch:
+  hosts: ["your-hostname:your-port"]
+  pipeline: dns_enrich_pipeline
+```
+
+#### 3. Refresh your packetbeat index
+
+You can optionally choose to refresh your packetbeat index mapping within Kibana:
+* navigate to `Stack Management > (Kibana) Index Patterns` 
+* select the applicable packetbeat index
+* click `refresh field list`
+
+#### 4. Verify enrichment fields
+
+Any packetbeat documents with the field `dns.question.registered_domain` should now have the enriched data:
+`ml_is_dga.*`
+
+
+## Experimental DGA ML Jobs and Rules
+
+Once packetbeat data is being enriched, there are some rules and ML jobs which can leverage the enriched fields. 
+The experimental rules and jobs will be staged separate from the model bundle under the [releases](https://github.com/elastic/detection-rules/releases) 
+as `ML-experimental-detections-YYYMMDD-N`.
+
+Note that if a rule is of `type = "machine_learning"`, then it may be dependent on a uploading and running a machine
+learning job first. If this is the case, it will likely be annotated within the `note` field of the rule.
+
+#### Uploading rules
+
+You can then individually upload these rules using the [kibana upload-rule](../CLI.md#uploading-rules-to-kibana) command
+
+#### Uploading ML Jobs
+
+Unzip released jobs and then run `python -m detection_rules es <args> experimental upload-ml-job <ml_job.json>`
+
+To delete a job, run `python -m detection_rules es <args> experimental delete-ml-job <job-name> <job-type>`
+
+Take note of any errors as some jobs may have dependencies on each other which may require stopping and or removing
+referenced jobs first.
+
+
+## For Maintainers
+
+### Validating release bundles and releasing
+
+Release assets are expected to be in certain formats with specific naming patterns and json structures.
+
+#### Filename patterns
+
+DGA model file naming convention should match the following patterns
+
+```json
+{
+  "model":                                "dga_*_model.json",
+  "dga_ngrams_create":                    "dga_*_ngrams_create.json",
+  "dga_ngrams_transform_delete":          "dga_*_ngrams_transform_delete.json",
+  "dns_enrich_pipeline":                  "dga_*_ingest_pipeline1.json",
+  "dns_dga_inference_enrich_pipeline":    "dga_*_ingest_pipeline2.json"
+}
+```
+
+Experimental detections do not have to match a specific naming pattern but should be in the following file formats:
+* rules: toml
+* jobs: json
+
+#### Uniqueness
+
+The model file name and hash should be unique or else it will raise a warning in validation. This is important to allow 
+distinction and ascertain information about a bundle by consulting the manifest, based on a unique name
+
+Release zipped assets, name, and tag name all share the same name. These should follow the following format:
+* Model releases: `ML-DGA-20\d\d[0-1]\d[0-3]\d-\d`
+* Detection releases: `ML-experimental-detections-20\d\d[0-1]\d[0-3]\d-\d`
```

Comment:
more obvious here:
```suggestion
* Model releases: `ML-DGA-YYYY-MM-DD`
* Detection releases: `ML-experimental-detections-YYYY-MM-DD`
```

Reply:
2 use cases that come to mind _for_ numbering:
* multiple releases in a day
* quickly ascertaining count / quick reference (_"Hey Ross, go checkout DGA release 51"_)